README.md

```diff
@@ -120,8 +121,19 @@ If you find an issue with this package and have a fix, please feel free to open
 
 ## Testing
 
-If you contribute to this package you can run the tests using `yarn test`. This package also includes a sample application for manual testing. First open a seperate terminal and run `yarn watch`, this will ensure the Typescript files in the src directory are compiled to Javascript in the lib directory. Navigate to `src/sample`, here you can edit `index.ts` to your liking and test your contributions manually. Run `npx cdk --app lib/sample/index.js <CDK Command>`, replacing `<CDK Command>` with common CDK commands like `synth`, `diff`, or `deploy`. 
-* Note, if you receive "... is not authorized to perform: ..." you may also need to authorize the commands with your AWS credentials.
+If you contribute to this package you can run the tests using `yarn test`. This package also includes a sample application for manual testing. First open a seperate terminal and run `yarn watch`, this will ensure the Typescript files in the src directory are compiled to Javascript in the lib directory. Navigate to `src/sample`, here you can edit `index.ts` to your liking and test your contributions manually. Run `npx cdk --app lib/sample/index.js <CDK Command>`, replacing `<CDK Command>` with common CDK commands like `synth`, `diff`, or `deploy`.
+
+- Note, if you receive "... is not authorized to perform: ..." you may also need to authorize the commands with your AWS credentials.
+
+### Debug Logs
+
+To display the debug logs for this library, set the `DD_CONSTRUCT_DEBUG_LOGS` env var to `true` when running `cdk synth`. 
+
+Example: 
+
+```
+DD_CONSTRUCT_DEBUG_LOGS=true cdk synth
```

Comment:
I noticed the cdk.json file was left out, technically if that file is not included in the codebase then I think the cdk commands will fail (at least this one on line 135) because it will say no stack was given. Im thinking having a cdk.json file would be beneficial in the long run because it shortens the commands. Maybe add the cdk.json file and once this PR is wrapped up I can make sure the readme advertises the shortened cdk commands.

Reply:
It's confusing to have both instructions because one is intended to run from the project root directory and the other from within `src/sample`. In #24, you can pull these changes in and update this command to:

`DD_CONSTRUCT_DEBUG_LOGS=true npx cdk --app lib/sample/index.js synth`